src/components/ResourceBrowser/ResourceList/ClusterSelector.tsx

```diff
@@ -36,18 +36,23 @@ const ClusterSelector: React.FC<ClusterSelectorType> = ({ onChange, clusterList,
     const defaultOption = filteredClusterList.find((item) => String(item.value) === clusterId)
 
     return (
-        <ReactSelect
-            classNamePrefix="cluster-select-header"
-            options={filteredClusterList}
-            onChange={onChange}
-            components={{
-                IndicatorSeparator: null,
-                DropdownIndicator,
-                LoadingIndicator: null,
-            }}
-            value={defaultOption}
-            styles={appSelectorStyle}
-        />
+        <div className="flexbox dc__align-items-center dc__gap-12">
+            <ReactSelect
```

Comment:
Why aren't we using the `<SelectPicker` ?

Reply:
Page Header ReactSelect 
we don't have the variant yet

<img width="453" alt="image" src="https://github.com/user-attachments/assets/bdf7f6b9-5168-4fc2-a51d-5e961992346c">